r/R/differential_expression.R

```diff
@@ -59,24 +59,36 @@ runDE <- function(req, data) {
 
   message("Paginating results:  ", str(result))
   pagination <- req$body$pagination
-  pathwayAnalysis <- req$body$pathwayAnalysis
+  genes_only <- FALSE
+  
   order_by <- pagination$orderBy
   order_decreasing <- pagination$orderDirection == "DESC"
   offset <- pagination$offset
   limit <- pagination$limit
   filters <- pagination$filters
 
+  if ("genesOnly" %in% names(pagination)) {
```

Comment:
I dont *love* that this is inside pagination but that's a tiny detail

Reply:
yea it's kinda wrong. I'll put it somewhere else

Reply:
Moved to body